### PR TITLE
pseudo element approach

### LINKED ISSUE #2168: Extensions categories have layout issues for RTL lang
Steps to reproduce:
1. Load Extensions categories page on your device https://addons-dev.allizom.org/en-US/android/extensions/categories/
2. Change the language to a RTL one i.e. https://addons-dev.allizom.org/ar/android/extensions/categories/

Expected results:
There are no display or layout issues.

Actual results:
Categories arrows are not RTL.

Notes/Issues:
Verified on AMO-dev FF52(Win 7).
Screenshot for this issue:

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #2168

### CODE REVIEW

**Comment by muffinresearch** [comment]
If you use generated content here: https://github.com/mozilla/addons-frontend/pull/2439/files#diff-0e3835cf21cbc3cf9ec95501830508b1L7 then you only need a transform for the rtl case.

**Comment by tsl143** [comment]
@muffinresearch  Removed background image from `.Categories-list-item` and added pseudo element on `.Categories-link`

File: src/amo/components/Categories.scss [hunk 1]
```diff
@@ -17,4 +16,17 @@
   display: block;
   padding: 1em 0;
   text-decoration: none;
+
+  &::before {
+    background: url('../img/icons/arrow.svg') 100% 50% no-repeat;
+    content: '';
+    float: right;
```
Comment (muffinresearch) [inline src/amo/components/Categories.scss]: I think if you set `display` to `inline-block` it might negate the need for floats.

**Comment by tsl143** [comment]
@muffinresearch But implementing `inline-block` will incur alignment (both vertical and left/right)

**Comment by muffinresearch** [comment]
> @muffinresearch But implementing inline-block will incur alignment (both vertical and left/right)

Yep, now I've spun it up and look at it in practice you're absolutely right. Sorry for the noise! This is ready to 🚢

**Comment by tsl143** [comment]
Awesome :)

**Comment by caitmuenster** [comment]
Woohoo! You're on fire this month, @tsl143! Thanks for your contributions!

**Pull request merged**